src/devices/bus/hp9845_io/hp9845_io.h

```diff
@@ -48,7 +48,7 @@ class hp9845_io_slot_device : public device_t,
 	virtual void device_start() override;
 };
 
-class hp9845b_state;
+class hp9845_base_state;
```

Comment:
This is a dependency from src/devices on src/mame, isn't it?  We really need to eliminate these things.  Please try to find a way to access the necessary resources through your slot interface.